main/explicit_bzero.c

```diff
@@ -32,6 +32,9 @@
 
 PHPAPI void php_explicit_bzero(void *dst, size_t siz)
 {
+#ifdef __NetBSD__
```

Comment:
Perhaps it would be better to put this condition before __GNUC__ instead of nesting it. It seems also, that a symbol check should be done, as NetBSD 6 seems to be still available.

Thanks.